pkg/util/apiserver/apiserver.go

```diff
@@ -108,13 +108,12 @@ func NewAPIServer(mux *http.ServeMux) *APIServer {
 // e.g. http://localhost:8001/apis/scheduling.k8s.io/v1beta1
 // as well as registering a CRDHandler that all http requests for the given APIResource are routed to
 func (as *APIServer) AddAPIResource(groupVersion string, resource metav1.APIResource, handler CRDHandler) {
-	list, ok := as.resourceList[groupVersion]
+	_, ok := as.resourceList[groupVersion]
```

Comment:
I'm not sure I understand the changes in this file. Should this be included? It doesn't have anything to do with fixing the lint errors.

Reply:
Removing this change from PR.